cumulusci/tasks/salesforce/tests/test_UpdateDependencies.py

```diff
@@ -116,6 +120,56 @@ def test_run_task(self, ApiRetrieveInstalledPackages):
         )
         self.assertEqual(10, task.api_class.call_count)
 
+    @mock.patch(
+        "cumulusci.salesforce_api.metadata.ApiRetrieveInstalledPackages.__call__"
+    )
+    def test_run_task_without_downgrades(self, ApiRetrieveInstalledPackages):
```

Comment:
Do we need a test case for a situation where `allow_newer` and `allow_uninstalls` are both `False`?

Reply:
Line coverage for the module was at 100%, but I added tests for all 4 combinations just to be sure.